### PR TITLE
LUN-214: Missing connection when adding discipline

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

Please test this issue on IOS!

Probably its not the best way to fix it on FE. We can discuss better ways
Maybe we should just check the Apache Logs.
Feel free to discuss

### CODE REVIEW

File: src/services/helpers.ts [hunk 1]
```diff
@@ -30,3 +30,8 @@ export const moveToEnd = <T>(array: T[], index: number): T[] => {
   newDocuments.push(currDocument)
   return newDocuments
 }
+
+// fix ios issue for Django, that requires trailing slash in request url https://github.com/square/retrofit/issues/1037
+export const addTrailingSlashToUrl = (url: string): string => {
+  return url.lastIndexOf('/') !== -1 ? url : `${url}/`
```
Comment (klinzo) [inline src/services/helpers.ts:36]: I don't think this works as expected. `'/asdf'.lastIndexOf('/')` also returns a value different to `-1` but has no trailing slash

**Review by klinzo — CHANGES_REQUESTED** [review]
I think this should be fixed in the backend

File: release-notes/unreleased/LUN-214-Missing-Connection-Adding-Discipline.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+issue_key: LUN-214
+show_in_stores: true
+platforms:
+  - ios
+de: Fehler beim Hinzufügen individueller Bereiche behoben
```
[the remark this answers is not in the document]
Comment (f1sh1918) [reply release-notes/unreleased/LUN-214-Missing-Connection-Adding-Discipline.yml]: removed

**Comment by f1sh1918** [comment]
> I think this should be fixed in the backend (at least permanently). As a short workaround this may be ok.

hm we can open a backend ticket, but as it looks its nothing to fix in a few minutes, so maybe we should maybe stick to this workaround and remove it lateron

**Review by SteffiStuffel — APPROVED** [review]
I cannot reproduce this when using Postman, both with and without slash works fine, not sure why this is. As this PR fixes a error we have, I would really appreciate if we can merge it soon.

**Comment by klinzo** [comment]
When does this happen? Cannot reproduce this on iOS simulator

**Pull request merged**